addons/survey/views/survey_survey_views.xml

```diff
@@ -170,7 +170,7 @@
         <field name="name">survey.survey.view.tree</field>
         <field name="model">survey.survey</field>
         <field name="arch" type="xml">
-            <tree string="Survey" sample="1">
+            <tree string="Survey" sample="1" js_class="survey_list">
```

Comment:
Would say survey_view_tree (keeping the reasoning from guidelines: model, action, type).